tests/unit_tests/db/test_db_handler.py

```diff
@@ -157,18 +158,19 @@ def test_copy_telescope_db(db, random_id, db_cleanup, io_handler, model_version)
 
 def test_add_tagged_version(db, random_id, db_cleanup, io_handler, model_version):
 
+    tags = {
+        "Released": {"Value": "2020-06-28"},
+        "Latest": {"Value": "2024-02-01"},
+        "Prod25": {"Value": "2020-06-28"},
+        "Prod26": {"Value": "2024-02-01"},
```

Comment:
I am a bit surprised to see Prod25 and Prod26, I thought we were in prod5, prod6 era.

Reply:
Yes, but I want to make sure that things are working with names other than prod5 or prod6 in the tests....